assets/home/components/LargeTextOnlyCard.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { gettext, shortDate, wordCount } from 'utils';
+import MoreNewsButton from './MoreNewsButton';
+
+const getTextOnlyPanel = (item, openItem) => (
+    <div key={item._id} className='col-sm-6 col-lg-4 d-flex mb-4'>
+        <div className='card card--home' onClick={() => openItem(item)}>
+            <div className='card-body'>
+                <h4 className='card-title'>{item.headline}</h4>
+                <div className='wire-articles__item__text'>
+                    <p>{item.description_text}</p>
+                </div>
+                <div className='wire-articles__item__meta'>
+                    <div className='wire-articles__item__meta-info'>
+                        <span className='bold'>{item.slugline}</span>
+                        <span>{gettext('Source: {{ source }}', {source: item.source})}
+                            {' // '} {shortDate(item.versioncreated)}
+                        </span>
+                    </div>
+                </div>
+            </div>
+            <div className='card-footer'>
```

Comment:
imo would be good to introduce some components for card parts, like CardFooter or CardBody seems like it's duplicated there